src/sbt-test/sbt-scalafix/resolvers/test

```diff
@@ -0,0 +1,7 @@
+# without the proper resolver/repository, invocation of the community rule fails
+-> scalafix SyntacticRule
+
+# once the proper resolver is setup via sbt, dependency fetching works fine and scalafix can be used normally
+> set ThisBuild / resolvers ++= Resolver.sonatypeOssRepos("snapshots")
```

Comment:
sorry, I mislead you on that one, but you should use the deprecated helper for compatiility with older sbt versions
```suggestion
> set ThisBuild / resolvers += Resolver.sonatypeRepo("snapshots")
```

Reply:
Nevermind :), changed